From 0759b0ec3252dc665043bff77e363b02e0937f5d Mon Sep 17 00:00:00 2001
From: David Plowman <david.plowman@raspberrypi.com>
Date: Tue, 25 Jul 2023 13:04:26 +0100
Subject: [PATCH 2/2] Fix a flake8 complaint

Signed-off-by: David Plowman <david.plowman@raspberrypi.com>
---
 apps/app_capture2.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/apps/app_capture2.py
+++ b/apps/app_capture2.py
@@ -9,7 +9,7 @@ from PyQt5.QtWidgets import (QApplicatio
                              QVBoxLayout, QWidget)
 
 from picamera2 import Picamera2
-from picamera2.previews.qt import QGlPicamera2, QPicamera2
+from picamera2.previews.qt import QGlPicamera2
 
 
 def post_callback(request):
